packages/components/src/Tabs/Tabs.tsx

```diff
@@ -78,6 +78,12 @@ export function Tabs({ children, defaultTab = 0, onTabChange }: TabsProps) {
     };
   });
 
+  useEffect(() => {
+    if (activeTabInitialValue !== activeTab) {
```

Comment:
Carly reconsider this case: the activeTab > React.Children.count(children)

Reply:
Updated to only call setActiveTab if activeTab is greater than the number of new tabs.